.stylelintrc.js

```diff
@@ -31,7 +32,7 @@ module.exports = {
             }
         ],
         'selector-max-id': 2,
-        'selector-max-compound-selectors': 5,
+        'selector-max-compound-selectors': 6,
```

Comment:
Bumped slightly to resolve a lot of issues and avoid having to add a ignore comments for all of them.
This rule can't be retrospectively respected without changing the specificity of selectors, which might affect styling in adverse ways.